### PR TITLE
Arxivce 2391 forum banners 2024

### PR DESCRIPTION
**Pull request by kyokukou**

### CODE REVIEW

File: browse/static/css/arXiv.css [hunk 1]
```diff
@@ -664,28 +664,47 @@ footer .sorry-app-links .a11y-main-link {
   justify-content:center;
 }
 .banner-minimal a {
-  -webkit-box-shadow: 0px 3px 6px 0px #000000);
+  -webkit-box-shadow: 0px 3px 6px 0px #000000;
```
Comment (norbusan) [inline browse/static/css/arXiv.css:667]: What a great change!!! Why was there a `)` in the beginning?

File: browse/static/css/arXiv.css [hunk 2]
```diff
@@ -664,28 +664,47 @@ footer .sorry-app-links .a11y-main-link {
...
   -moz-box-shadow: 0px 3px 6px 0px #000000;
   box-shadow: 0px 3px 6px 0px #000000;
   display: inline-block;
   padding: .25em 1em;
   border-radius: .3em;
   font-size: 2em;
   background-color: #1c1a17;
-  border: 2px solid #6b6459;;
+  border: 2px solid #6b6459;
```
Comment (norbusan) [inline browse/static/css/arXiv.css:675]: Wow, another great change

File: browse/templates/user_banner.html [hunk 3]
```diff
@@ -19,80 +19,99 @@
...
+    ((BANNER_START_2|int <= now) and (BANNER_END_2|int > now)) or
+    ((BANNER_START_3|int <= now) and (BANNER_END_3|int > now)) or
+    ((BANNER_START_4|int <= now) and (BANNER_END_4|int > now)) -%}
+    {# only run this script while a banner is active #}
+
+    <link rel="stylesheet" type="text/css" media="screen" href="{{ url_for('static', filename='css/slider.css') }}?v=1.1" />
+    <script src="//code.jquery.com/jquery-latest.min.js" type="text/javascript"></script>
+    <script type="text/javascript" src="{{ url_for('static', filename='js/donate.js') }}"></script>
+  {%- endif -%}
+{%- endmacro -%}
```
Comment (norbusan) [inline browse/templates/user_banner.html:117]: No comment on all the html changes

**Review by norbusan — COMMENTED** [review]
LGTM, but I have no idea about html/css in reality ...

File: browse/static/css/arXiv.css (hunk 1 above)
Comment (kyokukou) [reply browse/static/css/arXiv.css:667]: looks like a typo. I think not everyone who works on the CSS uses code editors that will aggressively highlight syntax errors

File: browse/static/css/arXiv.css (hunk 2 above)
Comment (kyokukou) [reply browse/static/css/arXiv.css:675]: what can I say, I'm just amazing ;)

**Review by jonathanhyoung — APPROVED** [review]

**Pull request merged**